lib/services/pages.js

```diff
@@ -282,6 +282,8 @@ function publish(uri, data, locals) {
         // make public
         addOpToMakePublic(ops, prefix, publicUrl, uri);
 
+        pageData.lastModified = new Date().toISOString();
```

Comment:
Not sure. I see two arguments for putting it here:

*  Amphora plugins, like `clay-sitemaps`, can expect it to be there.
* Either way, Amphora must make an exception for the `lastModified` key in `references.omitPageConfiguration`. Amphora hard-coding an exception like this because of logic outside of it is bad architecture, IMO. (This is what's going on with `urlHistory`).